thoth/common/openshift.py

```diff
@@ -1472,6 +1472,7 @@ def schedule_mi_workflow(
         knowledge_path: Optional[str] = None,
         mi_used_for_thoth: Optional[bool] = False,
         mi_merge: Optional[bool] = False,
+        mi_merge_path: Optional[str] = None,
```

Comment:
it can be None?

Reply:
yes, if `mi_merge = True`  then the default location will be used.
If it is `False`, then we don't care and it is None.

Reply:
@pacospace wdyt?